Assets/Scripts/UI/Dialogue/DialogueKeyWords.cs

```diff
@@ -0,0 +1,7 @@
+public enum DialogueKeyWords

```

Comment:
Waar deze Enum wordt gebruikt, zal later veranderd moet worden met de hele lijst die Bjorn zal maken. Of de naam kan beter.